site/gatsby-site/src/components/forms/SubmissionWizard/StepOne.js

```diff
@@ -115,10 +115,10 @@ const FormDetails = ({
   } = useFormikContext();
 
   useEffect(() => {
-    if (!values['date_downloaded']) {
+    if (!values['date_downloaded'] || values['date_downloaded'] === '') {
       setFieldValue('date_downloaded', new Date().toISOString().substr(0, 10));
     }
-  }, []);
+  }, [values['date_downloaded']]);
```

Comment:
If I don't use it here it will only execute on componentDidMount, and the `date_downloaded` doesn't get updated later on, once its value is set. And this is what happens:

https://user-images.githubusercontent.com/4105289/203329241-e6670569-c35f-42ef-b43a-8feea9f6ddc1.mov

Reply:
Sorry I meant that you could use `values.date_downloaded` instead. It makes code easier to read.